bookserver/schemas.py

```diff
@@ -61,6 +63,7 @@ class LogItem(LogItemIncoming):
     class Config:
         orm_mode = True
         # this tells pydantic to try read anything with attributes we give it as a model
+        # TODO: This doesn't make sense to me -- we're not using an ORM. What does this do?
```

Comment:
I don't recall.  It may be cruft some something I was experimenting with.  I don't see anywhere in  our code where we are using this Class.

Reply:
A little more research reminded me that this is how you configure Pydantic.    See:  https://pydantic-docs.helpmanual.io/usage/models/#orm-mode-aka-arbitrary-class-instances I'm not sure we want to remove it.  I think a little better understanding of how we are using Pydantic will make the decision.

Reply:
I saw that page, but don't see how it's relevant to what we're doing. My understanding of the approach:

1. Given a sqlalchemy table, auto-generate a Pydantic schema for it.
2. From an endpoint, put data from Ajax into the validator and check it before writing to the db. The validator will also handle conversion from string to other data types if this came as query params.

Reply:
So, we are not doing 1 yet.  But I think we agree on that as the direction.

For 2, I think the other point is that the Schema can define what we get from the client as well as validate.  Right now in web2py you really have no way of knowing what data might be in `request.vars`. Using one of these schema objects tells you immediately what you get.  But I think that means that we will end up creating additional schema objects that do not have a one to one mapping with the database.

Reply:
I agree -- creating additional schemas for things that don't map directly to a db table makes sense. The lack of any kind of validation in web2py code produced a lot of bugs/problems.

I mainly wanted to avoid duplicating schemas when the sqlalchemy Table structure already has that data (keep the code DRY and avoid mistakes where one place is updated but the other isn't.)